### PR TITLE
doc: Updated documentation for Zabbix Mgr module

### PR DESCRIPTION
**Pull request by wido**
Signed-off-by: Wido den Hollander <wido@42on.com>

### CODE REVIEW

File: doc/mgr/zabbix.rst [hunk 1]
```diff
@@ -47,8 +47,8 @@ Configuration
 
 Two configuration keys are mandatory for the module to work:
 
-- mgr/zabbix/zabbix_host
-- mgr/zabbix/identifier
+- zabbix_host
+- identifier
```
Comment (tchaikov) [inline doc/mgr/zabbix.rst]: [ot],

@jcsp @wido   the settings are stored using the prefix of `get_mgr_id() / ` (i.e. `$name/`), so there is chance that the config item name of one plugin could collide with those in another plugin. shall we update `{get,set}{_localized,}_config()` to address this by, for example, append the plugin name to the prefix?

File: doc/mgr/zabbix.rst [hunk 2]
```diff
@@ -72,14 +72,32 @@ these are usually Monitors where the *client.admin* key is present.
...
+
+The current configuration of the module can also be shown:
+
+::
+
+   ceph zabbix config-show
+
+Manually sending data
+---------------------
+If needed the module can be asked to send data immediately instead of waiting for
```
Comment (tchaikov) [inline doc/mgr/zabbix.rst]: nit, add a comma after "If needed".

**Comment by wido** [comment]
@tchaikov Yes, I understand. But when developing the module I understood that this was 'the way to go' for modules.

Users should not be setting the config-keys manually since it would require a manager restart.

**Comment by tchaikov** [comment]
@wido, when i am looking at the key name of "identifier", i fell that other plugin could also use it. it's way too general. thus the comment. i don't think my conern is about how user is supposed to set config, it's about how can we protect plugins from interfere with each other by sharing the config-key names.

**Comment by wido** [comment]
@tchaikov Ah yes, but it's still stored as a config-key under the namespace of the manager. Nothing changed in the Python code. It's just a doc change.

**Comment by tchaikov** [comment]
@wido, yeah. so that's but a [OT] (off-topic). i think we need to address it sooner than latter.

**Comment by wido** [comment]
@tchaikov Ok! Let's do that somewhere else

**Comment by tchaikov** [comment]
oh, never mind. `PyModules::set_config()` does use `handle` as part of the naming space of conf-key. sorry for the noise =(

**Pull request merged**